ingest/mongo_connection.py

```diff
@@ -34,6 +34,7 @@ def __init__(self):
                 password=self.password,
                 authSource=self.db_name,
                 authMechanism="SCRAM-SHA-1",
+                serverSelectionTimeoutMS=60_000
```

Comment:
For reference, we believe the [original timeout was 20 seconds](https://broadinstitute.slack.com/archives/CBEHTH601/p1701113814926589?thread_ts=1701113580.649469&cid=CBEHTH601).